src/hooks/useBalanceChartData/useBalanceChartData.test.ts

```diff
@@ -16,15 +16,15 @@ import {
 
 const mockedDate = '2021-11-20T00:00:00Z'
 
-const ethCaip19 = 'eip155:1/slip44:60'
-const foxCaip19 = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
+const ethAssetId = 'eip155:1/slip44:60'
+const foxAssetId = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
```

Comment:
This one is the way to go with a locally defined `AssetId` const however, we probably shouldn't abstract anything else than native tokens' `AssetId`s in caip exports or this would get really messy really fast